packages/react-instantsearch/src/core/createInstantSearch.test.js

```diff
@@ -1,100 +1,61 @@
 /* eslint-env jest, jasmine */
 /* eslint-disable no-console */
 import React from 'react';
-import {mount} from 'enzyme';
-
-import algoliaClient from 'algoliasearch';
+import {shallow} from 'enzyme';
```

Comment:
Was there another reason to switch to `shallow` apart the fact than a dom wasn't needed?